.github/workflows/pipeline.yml

```diff
@@ -5,7 +5,7 @@ on: [push]
 
 jobs:
   tests:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
Per the feedback on https://github.com/Invoca/invoca-utils/pull/24/files#diff-fe700949ba86403864aa7b6fdcab2c00c55e1047f73a6a282135a35e8a21e03b

Relaxing the specific version requirement so that there is no need to follow up
when the support for a specified version is dropped upstream or the latest version
is released.